README.md

```diff
@@ -15,6 +15,11 @@ The first time it searchs all folders, after that it uses a cached repository in
 
 ![open Git Project](/img/openProject.gif)
 
+### GPM: Open Git Project from SubFolder *(Defaults to: Shift+Alt+P)
+This commands open a git project from subfolder
+
+### GPM: Open Recent Git Project *(Defaults to: Ctrl+Shift+Q)
+This commands open a recent git project 
```

Comment:
It wold be nice to have here additional info regards a configuration that defines how many recent projects were added